src/chilean-chickpeas.md

```diff
@@ -0,0 +1,31 @@
+# Chilean Chickpeas
+
+Traditional chilean dish.
+- ⏲️ Prep / Cook time: 30 - 40 m
+- 🍽️ Servings: 6
+
+## Ingredients
+
+- Dry chickpeas: 500 g
+- Sweet potato squash: 250 g
+- Onion: 1
+- Carrot: 1 cup
+- Pepper: 1 cup
+- Celery: 1 cup
```

Comment:
Is the celery grated as well? Otherwise not sure how to measure celery in volume.